MyPerf4J-Base/src/test/java/cn/myperf4j/base/util/concurrent/AtomicIntHashCounterTest.java

```diff
@@ -162,7 +162,17 @@ public void testSingleThreadV2() {
 
     @Test
     public void testMultiThread4HighRace() throws InterruptedException, BrokenBarrierException {
-        final int threadCnt = Runtime.getRuntime().availableProcessors() - 2;
+        //final int threadCnt = Runtime.getRuntime().availableProcessors() - 2;
+        final int availableThreads = Runtime.getRuntime().availableProcessors();
+        int threadCnt;
+        if (availableThreads <= 1) {
+            throw new RuntimeException("Multithreading is not supported on this system with " +
+                    availableThreads + " available processors.");
+        } else if (availableThreads >= 2 && availableThreads <= 3) {
```

Comment:
Could you please elaborate on why you chose to assign threadCnt = 2 when there are 2 or 3 available processors?
I think the authors might have an issue if both processors get used since their primary motive was to keep 2 processors available for other computing tasks.
In case of 3 available threads, maybe we can assign just one thread since that is what the authors initially intended? (However, I'm not 100% sure if this is the best approach)